site/content/en/docs/Installation/Terraform/eks.md

```diff
@@ -64,9 +64,9 @@ kubectl get nodes
 
 ### Uninstall the Agones and delete EKS cluster
 
-Run the following commands to delete all Terraform provisioned resources if you choose helm:
+Run the following commands to delete all Terraform provisioned resources:
 ```
-helm delete --purge agones
+terraform destroy -target module.helm_agones.helm_release.agones -auto-approve && sleep 60
```

Comment:
QQ - I assume this is a stopgap until we can do #1422  ?

Reply:
Yes, it is. I also added this into `terraform.mk` and it cleans up all Firewall Rules correctly. To conclude I have tested this approach on GKE and EKS.